libs/client/my-stuff/src/lib/views/prose-form/prose-form.component.ts

```diff
@@ -83,8 +93,36 @@ export class ProseFormComponent implements OnInit {
     }
 
     submitForm(contentId?: string) {
-        if (this.proseForm.invalid) {
-            this.alerts.warn(`Looks like something's wrong with the stuff you've entered.`);
+        if (this.fields.title.invalid) {
+            this.alerts.warn('Title field has an invalid length.');
+            return;
+        }
+        if (this.fields.desc.invalid) {
+            this.alerts.warn('Short description has an invalid length.');
+            return;
+        }
+        if (this.fields.body.invalid) {
+            this.alerts.warn('Body text is too short.');
+            return;
+        }
+        if (this.fields.category.invalid) {
+            this.alerts.warn('Category is required.');
+            return;
+        }
+        if (this.fields.genres.invalid) {
+            this.alerts.warn('Invalid number of genres. Limit is 3.');
+            return;
+        }
+        if (this.fields.tags.invalid) {
+            this.alerts.warn('Invalid number of tags. Limit is 5.');
```

Comment:
Don't think this has a constant. And it's not enforced in the backend, right? We should probably do so--it's pretty easy to call these endpoints without touching the website at all.